test/test_fake_quant.py

```diff
@@ -256,6 +256,7 @@ def test_backward_per_channel(self, device, X):
     @given(device=st.sampled_from(['cpu', 'cuda'] if torch.cuda.is_available() else ['cpu']),
            X=hu.per_channel_tensor(shapes=hu.array_shapes(1, 5,),
            qparams=hu.qparams(dtypes=torch.quint8)))
+    @unittest.skip("temoprarily disable the test")
```

Comment:
NIT: temporarily

Reply:
Thanks for catching this! Also fixed the typo in another file.